ui/src/components/GroupTree/data.js

```diff
@@ -24,7 +24,7 @@ export const StageFields = [
   { label: 'TTL number', key: 'ttlNum' },
   { label: 'Segment interval unit', key: 'segmentIntervalUnit' },
   { label: 'Segment interval number', key: 'segmentIntervalNum' },
-  { label: 'Node selector', key: 'nodeSelector' },
+  { label: 'Stages', key: 'stages' },
```

Comment:
It is not "nodeSelector"; it should be "defaultNodeSelector." This term exists in the "ResourceOpt" of "Group," but does not exist in "Stage."

Reply:
hmm....Currently, there doesn't seem to be a `defaultNodeSelector` in UI. Will add directly the `stages ` here.